go/vt/sqlparser/sql.y

```diff
@@ -313,7 +312,7 @@ func bindVariable(yylex yyLexer, bvar string) {
 %token <str> SEQUENCE MERGE TEMPORARY TEMPTABLE INVOKER SECURITY FIRST AFTER LAST
 
 // Migration tokens
-%token <str> VITESS_MIGRATION CANCEL RETRY COMPLETE CLEANUP THROTTLE UNTHROTTLE EXPIRE RATIO
+%token <str> VITESS_MIGRATION CANCEL RETRY LAUNCH COMPLETE CLEANUP THROTTLE UNTHROTTLE EXPIRE RATIO
```

Comment:
Could you add LAUNCH to the non_reserved_keywords list as well? Otherwise statements like `create tablet launch (id int)` will stop working.

Reply:
Tested it out too. It returns an error like `Code: INVALID_ARGUMENT syntax error at position 20 near 'launch'`

Reply:
Done!